version/version.go

```diff
@@ -4,5 +4,5 @@
 
 package version
 
-// Version describes the version of build
-var Version = "0.0.1"
+// Version describes the version of Shipwright build controller
+var Version = "devel"
```

Comment:
I don't think we are overwriting this value in `make build` or `make release`. If not, we should do so via go flags.

Reply:
Agree.

I changed it as a go flags in our main.go, now it can be set in the different release deployment, like:
```
➜  build git:(operator_refine) ✗ go run cmd/manager/main.go --version v0.3.0
{"level":"info","ts":1614919747.8687959,"logger":"build","msg":"Shipwright Build Controller Version: v0.3.0"}
{"level":"info","ts":1614919747.868852,"logger":"build","msg":"Go Version: go1.14.13"}
{"level":"info","ts":1614919747.868873,"logger":"build","msg":"Go OS/Arch: darwin/amd64"}
```

At the same time, I also remove the `operator-sdk` version in the **printVersion** func.

Thanks!